app/assets/javascripts/components/generic_object/main-custom-button-form-component.js

```diff
@@ -125,13 +125,32 @@ function mainCustomButtonFormController(API, miqService, $q, $http) {
   };
 
   vm.saveClicked = function() {
+    miqService.sparkleOn();
```

Comment:
..and we probably need `miqService.miqFlashClear();` in both saveClicked and addClicked.

Because otherwise:

1. fail the validation with duplicate name
  => Name has already been taken appears

2. fix the name
3. fail the validation with duplicate desciption
  => Desciption has already been taken appears

4. you're seeing both flash messages, but one is obsolete now

Reply:
(or you could just replace `add_flash` with `miqService.miqFlash`, but that would not work when both are supposed to be shown, so you'd have to merge the messages into one)